flixel/input/FlxSwipe.hx

```diff
@@ -3,60 +3,140 @@ package flixel.input;
 import flixel.FlxG;
 import flixel.math.FlxMath;
 import flixel.math.FlxPoint;
+import flixel.util.FlxPath;
 import flixel.util.FlxStringUtil;
 
 @:allow(flixel.input.mouse.FlxMouseButton)
 @:allow(flixel.input.touch.FlxTouch)
 class FlxSwipe
 {
+	public static var VELOCITY_THRESHOLD = 100;
+	
+	public static function filterSwipes(swipes:Array<FlxSwipe>)
+	{
+		for (swipe in swipes) 
+		{
+			if (swipe.released)
+			{
+				swipe.destroy();
+				swipes.remove(swipe);
+			}
+		}
+	}
+	
 	/**
 	 * Either LEFT_MOUSE, MIDDLE_MOUSE or RIGHT_MOUSE, 
 	 * or the touchPointID of a FlxTouch.
 	 */
 	public var ID(default, null):Int;
 	
-	public var startPosition(default, null):FlxPoint;
-	public var endPosition(default, null):FlxPoint;
+	public var path(default, null):FlxPath;
```

Comment:
Creative use of `FlxPath`. Wouldn't a plain `Array<FlxPoint>` be more suitable though?